docs/root/configuration/http/http_filters/rbac_filter.rst

```diff
@@ -11,6 +11,10 @@ as well as the incoming request's HTTP headers. This filter also supports policy
 and shadow mode, shadow mode won't effect real users, it is used to test that a new set of policies
 work before rolling out to production.
 
+When a request is denied, the access log will set the flag `RBAC`, this helps to distinguish the
```

Comment:
Super nit, this part: "the access log will set the flag `RBAC`"

should be: access log response flag segment? (or similar)